CHANGES

```diff
@@ -41,6 +41,10 @@
      necessary to configure just to create a source distribution.
      [Richard Levitte]
 
+  *) Add option to build devcrypto egine as a dynamic engine.  Enable by
+     configuring with enable-devcryptoeng-dynamic.
+     [Eneas U de Queiroz]
+
```

Comment:
This should move up :wink:

Reply:
Hopefully, I won't have to do many rebases.  BTW, are you jumping versions from 1.1.1 to 3?

Reply:
Next major release will be 3.0.0, and we're changing versioning scheme.  Have a look at our [blog entry](https://www.openssl.org/blog/blog/2018/11/28/version/) on the subject.